homeassistant/components/lifx/light.py

```diff
@@ -609,7 +609,7 @@ async def set_state(self, **kwargs):
                 if power_off:
                     await self.set_power(ack, False)
                 if hsbk:
-                    await self.set_color(ack, hsbk, kwargs)
+                    await self.set_color(ack, hsbk, kwargs, duration=fade)
                 if power_on:
                     await self.set_power(ack, True, duration=fade)
```

Comment:
What happens if `power_on` is `True`? It is supposed to fade from black to the new color but I guess with this change it will start from whatever the color was before?

Reply:
@amelchio I just tested it, and the bulb can fade color while it is fading power, and that is what happens.

We could switch it to check for `hsbk and power_on` and then use the old behavior then. That seems reasonable.

Alternatively, a user could emulate the old behavior by first changing the color without transition and then turning on the lamp with transition.

I'm think either way is fine.